.ci/static.sh

```diff
@@ -1,45 +1,51 @@
 #!/usr/bin/env bash
+if [[ ! -e .ci/common.sh || ! -e nengo ]]; then
+    echo "Run this script from the root directory of this repository"
+    exit 1
+fi
+source .ci/common.sh
+
+# This script runs the static style checks
 
 shopt -s globstar
 
 NAME=$0
 COMMAND=$1
 STATUS=0  # Used to exit with non-zero status if any check fails
 
-function usage {
-    echo "usage: $NAME <command>"
-    echo
-    echo "  install  Install dependencies for running static checks"
-    echo "  run      Run static checks"
-    exit 1
-}
-
 if [[ "$COMMAND" == "install" ]]; then
-    conda install jupyter
-    pip install codespell flake8 pylint
-elif [[ "$COMMAND" == "run" ]]; then
+    # pip installs a more recent entrypoints version than conda
```

Comment:
We should check if this is still necessary, or if conda has updated their `entrypoints` version now.

Reply:
I did a quick check, and it appears they're both 0.3 now.

Reply:
Cool, I'd just take this out then to keep things simple